app/mm-redux/actions/users.ts

```diff
@@ -724,64 +692,6 @@ export function getSessions(userId: string): ActionFunc {
     });
 }
 
-export function revokeSession(userId: string, sessionId: string): ActionFunc {
-    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
-        try {
-            await Client4.revokeSession(userId, sessionId);
-        } catch (error) {
-            forceLogoutIfNecessary(error, dispatch, getState);
-            dispatch(logError(error));
-            return {error};
-        }
-
-        dispatch({
-            type: UserTypes.RECEIVED_REVOKED_SESSION,
-            sessionId,
-            data: null,
-        });
-
-        return {data: true};
-    };
-}
-
-export function revokeAllSessionsForUser(userId: string): ActionFunc {
-    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
-        try {
-            await Client4.revokeAllSessionsForUser(userId);
-        } catch (error) {
-            forceLogoutIfNecessary(error, dispatch, getState);
-            dispatch(logError(error));
-            return {error};
-        }
-        const data = {isCurrentUser: userId === getCurrentUserId(getState())};
-        dispatch(batchActions([
-            {
-                type: UserTypes.REVOKE_ALL_USER_SESSIONS_SUCCESS,
-                data,
-            },
-        ]));
-
-        return {data: true};
-    };
-}
-
-export function revokeSessionsForAllUsers(): ActionFunc {
-    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
-        try {
-            await Client4.revokeSessionsForAllUsers();
-        } catch (error) {
-            forceLogoutIfNecessary(error, dispatch, getState);
-            dispatch(logError(error));
-            return {error};
-        }
-        dispatch({
-            type: UserTypes.REVOKE_SESSIONS_FOR_ALL_USERS_SUCCESS,
-            data: null,
-        });
-        return {data: true};
-    };
-}
-
 export function loadProfilesForDirect(): ActionFunc {
```

Comment:
I'm not sure if we are using the `loadProfilesForDirect` method

Reply:
I think we are.. but if we aren't we can avoid it when porting to v2